v21.2/cockroach-demo.md

```diff
@@ -11,6 +11,7 @@ The `cockroach demo` [command](cockroach-commands.html) starts a temporary, in-m
 - By default, `cockroach demo` starts in secure mode using TLS certificates to encrypt network communication. It also serves a local [DB Console](#connection-parameters) that does not use TLS encryption.
 - Each instance of `cockroach demo` loads a temporary [Enterprise license](https://www.cockroachlabs.com/get-cockroachdb) that expires after an hour. To prevent the loading of a temporary license, set the `--disable-demo-license` flag.
 -  `cockroach demo` opens the SQL shell with a new [SQL user](authorization.html#sql-users) named `demo`. The `demo` user is assigned a random password and granted the [`admin` role](authorization.html#admin-role).
+-  <span class="version-tag">New in v21.2:</span> The <a href="#global-flag">`--global`</a> flag is no longer experimental. This flag simulates realistic global latencies in a multi-region cluster by setting the [`--locality` flag on node startup](cockroach-start.html#locality) to three different regions.
```

Comment:
IMO, we don't need this bullet.

If you disagree and want to keep it, we should probably make it read more like this:
```
The <a href="#global-flag">`--global`</a> flag simulates realistic global latencies in a multi-region cluster by setting the [`--locality` flag on node startup](cockroach-start.html#locality) to three different regions.
```

Reply:
Removing sgtm, wasn't sure about it. Should we have a `new in version` tag in the table of flags?

Reply:
> Should we have a new in version tag in the table of flags?

Since the flag already existed in 21.1, probably not. Just removing the experimental designation is probably enough.